Library/Formula/cowsay.rb

```diff
@@ -1,8 +1,8 @@
 class Cowsay < Formula
   desc "Configurable talking characters in ASCII art"
   homepage "https://web.archive.org/web/20120225123719/http://www.nog.net/~tony/warez/cowsay.shtml"
-  url "http://ftp.acc.umu.se/mirror/cdimage/snapshot/Debian/pool/main/c/cowsay/cowsay_3.03.orig.tar.gz"
-  sha256 "0b8672a7ac2b51183780db72618b42af8ec1ce02f6c05fe612510b650540b2af"
+  url "https://ftp.acc.umu.se/mirror/cdimage/snapshot/Debian/pool/main/c/cowsay/cowsay_3.03+dfsg2.orig.tar.gz"
+  sha256 "3b89965c7d6b19f321867e59d14d4aec820d36068f56d2b1e783498beeb4183e"
```

Comment:
Looking at the install message, it looks like it may have misparsed this to include the `.orig` as part of the version number. Probably worth adding an explicit `version "3.0.3+dfsg2"` after `sha256`.
```suggestion
  sha256 "3b89965c7d6b19f321867e59d14d4aec820d36068f56d2b1e783498beeb4183e"
  version "3.0.3+dfsg2"
```